sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -85,7 +84,7 @@ internal sealed class ServiceBusListener : IListener, IScaleMonitorProvider, ITa
             _isSessionsEnabled = isSessionsEnabled;
             _autoCompleteMessages = autoCompleteMessages;
             _triggerExecutor = triggerExecutor;
-            _cancellationTokenSource = new CancellationTokenSource();
```

Comment:
We still need the token source to handle the potential of calling Dispose directly without calling stop first (I know this doesn't happen today, but it could change).